lib/hanami/controller.rb

```diff
@@ -13,19 +13,6 @@ module Hanami
   # @since 0.1.0
   #
   # @see Hanami::Action
-  #
-  # @example
-  #   require "hanami/controller"
-  #
-  #   module Articles
-  #     class Index < Hanami::Action
-  #       # ...
-  #     end
-  #
-  #     class Show < Hanami::Action
-  #       # ...
-  #     end
-  #   end
```

Comment:
This example was weird - as it does not show anything about `Hanami::Controller`, I decided to delete it